doc/api/cli.md

```diff
@@ -181,6 +181,13 @@ added: v9.0.0
 
 Specify the `file` of the custom [experimental ECMAScript Module][] loader.
 
+### `--max-http-header-size=size`
+<!-- YAML
+added: REPLACEME
+-->
+
+Specify the maximum size, in bytes, of HTTP headers. Defaults to 8KB.
```

Comment:
perhaps make more clear that it effects all HTTP, that it can't be changed at runtime, and that setting it larger than necessary can allow DoS attacks to be more effective?